GUI/src/javafxgui/JavaFXComponents/GUIView.java

```diff
@@ -2,17 +2,16 @@
 
 import javafx.scene.Node;
 import javafx.scene.Scene;
-import javafxgui.controller.Controller;
+import javafxgui.controller.GUIAppController;
 import jttt.core.board.Board;
 import jttt.core.board.Mark;
 import jttt.core.game.GameType;
-import jttt.core.players.Player;
 
 public class GUIView {
 
     private final int DEFAULT_BOARD_DIMENSION = 3;
     private Scene scene;
-    private Controller controller;
+    private GUIAppController controller; // IS THIS OKAY TO DO SINCE IT'S UNLIKELY ANY OTHER GUIAPP COULD BE USED ?
```

Comment:
By doing this I can use a GUIAppController only method and won't have to cast the controller at that point.  I COULD I suppose extract that single method into it's own Interface, but I wasn't sure that was acceptable for a single method?